src/platform_linux/laio.c

```diff
@@ -292,21 +380,35 @@ laio_cleanup(io_handle *ioh, uint64 count)
    int             status;
 
    io = (laio_handle *)ioh;
+
+   // Check for completion of up to 'count' events, one event at a time.
+   // Or, check for all outstanding events (count == 0)
```

Comment:
No code changes in this busy diff block. Added comments, and elaborated data printed when assert fails, below.